WooCommerce/Classes/ViewModels/Booking Details/BookingAttendanceStatus+Localization.swift

```diff
@@ -13,7 +13,7 @@ extension BookingAttendanceStatus {
         case .checkedIn:
             return NSLocalizedString(
                 "BookingAttendanceStatus.checkedIn",
-                value: "Checked In",
+                value: "Checked-in",
```

Comment:
Instead of manually updating the strings in `Localizable.strings‎`, you can update the key for the updated string. The new key and value will be generated automatically in `Localizable.strings‎`.


```suggestion
                "bookingAttendanceStatus.checkedIn.title",
                value: "Checked-in",
```

Similar suggestion for the other change below.

Reply:
I see, but changing the key would mean that the intention of the text is different, but this is not the case now. Just the text is changing. In that sense, I'd keep the key and only change the value.

Reply:
Yeah I get your point, you might not need new translation for this text. I'm good with keeping this as-is.